cfgov/cfgov/settings/base.py

```diff
@@ -152,6 +152,11 @@
 
 ALLOWED_HOSTS = ['*']
 
+#Enabling compression for use in base.html
+COMPRESS_ENABLED = True
```

Comment:
So a few things to consider here: The JS is already being compressed at build-time (see the before PR screenshot) by webpack, so this adds the concatenation and caching of the JS file, with the consequence that the compression is being applied to an already compressed JS file at runtime (instead of build-time) and is creating a separate request for the JS file. I'm not an expert on caching, but if the page itself (with inline compressed JS) is cached, it may be less performant to run compression again and have a separate request for the JS. However, the page content will likely be updated more often than the JS so it does make sense to separate it out. 
On the local.py question, if `DEBUG` is `True`, the docs seem to indicate that `COMPRESS_ENABLED` would be cancelled out.
I suspect webpack will be more flexible in how it handles the JS compression, and it also happens at build-time. So alternatively it might make sense to have webpack concatenate the common.js and page-specific JS files instead and place them at a URL that's loaded in. However, this depends on what django does in regard to caching. Can we place JS at a URL and have django do caching magic without further compression?